browser/src/core/Socket.js

```diff
@@ -1158,10 +1158,9 @@ app.definitions.Socket = L.Class.extend({
 			// Switching modes.
 			window.location.reload(false);
 		}
-		else if (textMsg.startsWith('tooltip: ')) {
-			var tooltipInfo = JSON.parse(textMsg.substring(textMsg.indexOf('{')));
-			this._map.uiManager.showDocumentTooltip(tooltipInfo, $('.leaflet-layer'));
-		}
+		//else if (textMsg.startsWith('tooltip: ')) {
```

Comment:
please remove it completely, showDocumentTooltip was changed so it might be confusing that uncommenting will work